packages/axelar-std/tests/test.rs

```diff
@@ -134,3 +134,105 @@ mod upgradable {
         assert_auth!(owner, client.migrate(&()));
     }
 }
+
+mod into_event {
+    use soroban_sdk::{Address, String};
+    use stellar_axelar_std::events::{fmt_last_emitted_event, Event};
+    use stellar_axelar_std_derive::IntoEvent;
```

Comment:
```suggestion
    use stellar_axelar_std::IntoEvent;
```
avoid importing directly from the derive crate

Reply:
Also addressed this in `IntoEvent` doc